keycloak/realm.go

```diff
@@ -108,5 +113,34 @@ func (keycloakClient *KeycloakClient) ValidateRealm(realm *Realm) error {
 		return fmt.Errorf("validation error: theme \"%s\" does not exist on the server", realm.EmailTheme)
 	}
 
+	if realm.InternationalizationEnabled == false && len(realm.SupportLocales) > 0 {
+		return fmt.Errorf("validation error: SupportLocales cannot be set if InternationalizationEnabled is false")
+	}
+
+	if realm.InternationalizationEnabled == false && len(realm.DefaultLocale) > 0 {
+		return fmt.Errorf("validation error: DefaultLocale cannot be set if InternationalizationEnabled is false")
+	}
```

Comment:
Are these two error messages still needed?  Based on the provider code, it looks like these simply can't be set without specifying the `internationalization {}` set.

Reply:
I have removed these two.